### PR TITLE
Added GitHub action ci

### LINKED ISSUE #6: Setup CI
So I've actually only used Travis CI in the past, but I'd be up for trying out GitHub Actions in this repo.

If anyone's got experiance here, and can provide a quick first pointer on how the GitHub actions file might need to look to start docker-compose, and run the test suite, then that'd be fab. Otherwise I'll try to take a dig sometime.

### PR DESCRIPTION
**Pull request by hramezani**
Fixes #6.

I used [GitHub action services](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices) instead of `docker-compose`.

### CODE REVIEW

**Comment by rafalp** [comment]
Could you please implement Tom's feedback to your PR so it follows conventions from other encode projects and also implements code coverage? Thanks!

**Comment by hramezani** [comment]
Sure! I will do them and will let you know.

**Comment by hramezani** [comment]
@rafalp I made it like other encode project(httpx) as Tom mentioned.
I just disabled the docs build because we don't have any doc for this project.

Please take a look and let me if there is anything else to do.

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,7 @@ test.db
 .coverage
 .pytest_cache/
 .mypy_cache/
-starlette.egg-info/
+broadcaster.egg-info/
 venv/
+build/
+dist/
```
Comment (tomchristie) [inline .gitignore:9]: I don't think the `.gitignore` should need to change as part of this pull request?

File: scripts/docs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
```
Comment (rafalp) [inline scripts/docs:1]: This file seems useless considered we don't have any docs for the lib.

File: broadcaster/_backends/base.py [hunk 3]
```diff
@@ -1,8 +1,10 @@
-import typing
+from typing import Any
+
+from broadcaster._base import Event
```
Comment (rafalp) [inline broadcaster/_backends/base.py:3]: This import should be relative.

File: broadcaster/_backends/memory.py [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import asyncio
 import typing
-from .base import BroadcastBackend
-from .._base import Event
+
+from broadcaster._backends.base import BroadcastBackend
```
Comment (rafalp) [inline broadcaster/_backends/memory.py:4]: Those imports should be relative.

File: broadcaster/_backends/postgres.py [hunk 5]
```diff
@@ -1,7 +1,10 @@
 import asyncio
+from typing import Any
+
 import asyncpg
-from .base import BroadcastBackend
-from .._base import Event
+
+from broadcaster._backends.base import BroadcastBackend
```
Comment (rafalp) [inline broadcaster/_backends/postgres.py:6]: Those imports should be relative.

File: broadcaster/_backends/redis.py [hunk 6]
```diff
@@ -1,8 +1,10 @@
-import asyncio_redis
 import typing
 from urllib.parse import urlparse
-from .base import BroadcastBackend
-from .._base import Event
+
+import asyncio_redis
+
+from broadcaster._backends.base import BroadcastBackend
```
Comment (rafalp) [inline broadcaster/_backends/redis.py:6]: Those imports should be relative,

**Review by rafalp — CHANGES_REQUESTED** [review]

File: scripts/docs (hunk 2 above)
Comment (hramezani) [reply scripts/docs]: I've removed this script

File: broadcaster/_backends/base.py (hunk 3 above)
Comment (hramezani) [reply broadcaster/_backends/base.py]: Done!

File: broadcaster/_backends/memory.py (hunk 4 above)
Comment (hramezani) [reply broadcaster/_backends/memory.py]: Done!

File: broadcaster/_backends/postgres.py (hunk 5 above)
Comment (hramezani) [reply broadcaster/_backends/postgres.py]: Done!

File: broadcaster/_backends/redis.py (hunk 6 above)
Comment (hramezani) [reply broadcaster/_backends/redis.py]: Done!

**Comment by hramezani** [comment]
@rafalp I addressed your comments.

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Nice work, thank you!

**Pull request merged**